output-blocks/src/Control/OutputCapable/Blocks/Type.hs

```diff
@@ -86,6 +89,20 @@ getOutputSequenceWithRating
   -> m (Maybe Rational, [Output])
 getOutputSequenceWithRating = Generic.getOutputSequenceWithRating English
 
+{-|
+Converts 'OutputCapable' value using 'GenericOutput'
+into a result and a list of 'Output'
+
+Consider using 'getOutputSequenceWithRating'
+or even more specific versions of 'Control.OutputCapable.Blocks.Type'
```

Comment:
This also makes only sense in `Control.OutputCapable.Blocks.Generic.Type` but not here inside `Control.OutputCapable.Blocks.Type`?